source/ways-of-working/pull-requests.html.md.erb

```diff
@@ -0,0 +1,158 @@
+---
+
+# <%= current_page.data.title %>
+
+As a PR writer I will:
+
+- keep PRs small
+
+As a PR reviewer I will:
+
+- make an effort to review as soon as I get a second.
+- approve, as long as it’s better than before
+- favour suggestions over rejections, particularly when a multipart PR is indicated
+
+We practise code review on all changes at HMCTS to:
+
+- improve code quality
+- get team buy in on changes
+- validate change correctness
+- knowledge sharing
+
+Pull requests should not be opened when everything is finished.
+They should be opened as soon as you have something to show.
+
+Leaving it to the last minute leaves a lot of pressure to the reviewer, 
+often someone is told the code is done but they aren’t 
+told that the review hasn’t happened, pressure comes on to ship the code but there’s design problems
+ in it that need addressing, this could take hours, or days.
+
+## Size
+The smaller your pull request the easier it is to review, to test and the safer it is to send to production.
+Separating your changes out is a great skill to practice, some tips:
+
+* don’t make unrelated changes in a PR, you should still make the change if you think it improves the code
+Just take the 10 minutes to create a new branch and open a separate PR, it can be quickly merged and you can continue with your task
+* refactor the code to make your change easier first
+* avoid unrelated whitespace changes if possible, they distract the reviewer and can cause conflicts with other people working in similar areas
+* look at the code before you start coding and see roughly what you need to do, perhaps do this with another developer, this can also make the review easier that on
+    if two developers were involved in the design there’s more buy in
+* mention in your PR description that there is going to be more PRs and what is still to come
+
+## When should review happen
+Reviews need to happen often, otherwise you will build up a backlog of changes.
+Before you start your task for the day check all the open PRs to see if there’s any code that needs a review or if someone needs a bit of help.
+The daily reminder can be automated by slack bot
+
+```
+/remind #my-pr-channel every weekday at 8:30am Why not start the day with coffee and reviewing code? :wink: Link in channel topic
+```
+But importantly reviews should not happen just first thing in the morning, 
+they should be done after every break, before you start a new ticket or after you’ve opened a PR yourself.
+
+Once you make your review comments especially if you block the merge you need to be ready to respond quickly to follow up comments so as not to hold up the person who has created the PR.
+
+## Who should review
+
+All developers in the team should be reviewing, junior developers to tech lead.
+Code review isn’t just about improving the code and the solution being correct, it is also about 
+sharing knowledge of the code and different techniques that can be used in software engineering.
```

Comment:
Not the best way , but its also worth tagging specific people to bring in their attention when you know they have expertise in a place where you are changing.

Reply:
done